restbridge/RestBridge.cpp

```diff
@@ -33,128 +34,126 @@
 
 namespace restbridge
 {
-
-static const std::string PUBLISHER_SCHEMA_SUFFIX = "cmd://";
-static const std::string SUBSCRIBER_SCHEMA_SUFFIX = "resp://";
+namespace
+{
+static const std::string DEFAULT_SCHEME = "hbp";
+static const uint16_t DEFAULT_PORT = 4020;
```

Comment:
Default port could be zero so that the operating system automatically selects an available one. That said, the port would have to be exposed in some way (Log?)